pages/datasets/_datasetId.vue

```diff
@@ -800,6 +796,17 @@ export default {
     }
   },
 
+  watchQuery(newQuery) {
```

Comment:
This function is causing an error, and I believe it's because `this` isn't available in scope. I believe this function should return Boolean, which would then trigger component's methods.
> Watch query strings and execute component methods on change (asyncData, fetch(context), validate, layout, ...)
https://nuxtjs.org/docs/2.x/components-glossary/pages-watchquery/

Perhaps using a watcher for the specific query parameters like `tab` and `path`. 

Screencap of what I'm experiencing:

https://user-images.githubusercontent.com/1493195/115880721-07fb9d00-a419-11eb-8274-6171df9a0827.mp4